.github/scripts/format-check.sh

```diff
@@ -12,7 +12,7 @@ unformatted() {
   return 0
 }
 
-source /tmp/conda/etc/profile.d/conda.sh
+source $(dirname ${BASH_SOURCE[0]})/common.sh
+source $CONDADIR/etc/profile.d/conda.sh
```

Comment:
Does it make sense to add this full path to conda.sh to the common file, too? It might reduce some duplication since we have this new file that helps with that.

Reply:
Good idea, will do.